### ISSUE TITLE
Iron fence / bars connect to Buildcraft tanks

### ISSUE THREAD

**Issue by MatthiasMann** [BuildCraft/BuildCraft#4427]
BuildCraft version: buildcraft-all-7.99.24.1
Forge version: forge-1.12.2-14.23.5.2838
Link to crash report or log: doesn't crash
Singleplayer or multiplayer: Singleplayer
Steps to reproduce: 
1: Place a build craft tank
2: Place iron fences (minecraft:iron_bars)  around the tank
3: See that the iron bars / fence connect connect to the tank as if it was a solid block

Additional information: 
It would be nice to surround the tanks with fences to guard a tank against (accidental) interaction or to give it a more industrial look.

**Comment by CrazyBonze** [comment]
Is this what you are talking about?
![image](https://user-images.githubusercontent.com/7456150/62845242-4ec45600-bc7b-11e9-9fe8-c395591e10f6.png)

I might give this a crack to get my Forge modding environment set up if someone thinks that this is a bug worth fixing. I am not much of a Java developer though, more C++ and Python so it might take me some tome to follow some guides and read some documentation and figure out what needs to be done.

**Comment by MatthiasMann** [comment]
Yeah, I think it's related to the side solid or shape information.
